robot_names: 
  - 'husky1'
  - 'husky2'
  - 'husky3'
  - 'husky4'
  - 'telemax1'